### PR TITLE
Docs grammar

### PR DESCRIPTION
**Pull request by tfurf**
Some simple grammar corrections in the first level docs.

## Thanks for contributing a pull request!

If you are contributing for the first time or provide a trivial fix don't worry too
much about the checklist - we will help you get started.

## Contribution checklist:

(also see [CONTRIBUTING.rst](../tree/master/CONTRIBUTING.rst) for details)

- [X] wrote descriptive pull request text
- [ ] added/updated test(s)
- [X] updated/extended the documentation
- [ ] added relevant [issue keyword](https://help.github.com/articles/closing-issues-using-keywords/) in message body
  * :shrug: No issue submitted.
- [X] added news fragment in [changelog folder](../tree/master/docs/changelog)
  * fragment name: `<issue number>.<type>.rst` for example (588.bugfix.rst)
  * `<type>` is must be one of `bugfix`, `feature`, `deprecation`,`breaking`, `doc`, `misc`
  * if PR has no issue: consider creating one first or change it to the PR number after creating the PR
  * "sign" fragment with "by :user:`<your username>`"
  * please use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files - by :user:`superuser`."
  * also see [examples](../tree/master/docs/changelog)
- [ ] added yourself to `CONTRIBUTORS` (preserving alphabetical order)
  * :shrug: not necessary here.

### CODE REVIEW

File: docs/config.rst [hunk 1]
```diff
@@ -190,9 +190,9 @@ environment.
 
 Settings defined in the top-level ``testenv`` section are automatically
 inherited by individual environments unless overridden. Test environment names
-can consist of alphanumeric characters and dashes; for example:
-``py38-django30``. The name will be split on dashes into multiple factors,
-meaning ``py38-django30`` will be split into two factors: ``py38`` and
+can consist of alphanumeric characters and dashes, for example
```
Comment (asottile) [inline docs/config.rst:193]: this is a comma splice -- the left hand side is correct here

File: docs/config.rst [hunk 2]
```diff
@@ -190,9 +190,9 @@ environment.
...
+``py38-django30``. The name will be split by dashes into multiple factors,
```
Comment (asottile) [inline docs/config.rst:194]: I think the left side makes more sense to me

File: docs/config.rst [hunk 3]
```diff
@@ -190,9 +190,9 @@ environment.
...
+meaning ``py38-django30`` will be split into two factors, ``py38`` and
```
Comment (asottile) [inline docs/config.rst:195]: this is now a comma splice, the left hand side is correct here

File: docs/config.rst [hunk 4]
```diff
@@ -206,13 +206,13 @@ various versions and implementations of Python and provide default values for
 - ``jython``: configures ``basepython = jython``
 
 It is also possible to define what's know as *generative names*, where an
-individual section maps to multiple environments; for example:
-``py{37,38}-django{30,31}``, which would generate four environments, each
-consisting of two factors a piece: ``py37-django30`` (``py37``, ``django30``),
+individual section maps to multiple environments. For example,
+``py{37,38}-django{30,31}``  would generate four environments, each
```
Comment (asottile) [inline docs/config.rst:210]: the whitespace here is incorrect

File: docs/config.rst (hunk 1 above)
Comment (tfurf) [reply docs/config.rst]: Reverted in latest.

File: docs/config.rst (hunk 2 above)
Comment (tfurf) [reply docs/config.rst]: Reverted in latest.

File: docs/config.rst (hunk 3 above)
Comment (tfurf) [reply docs/config.rst]: Reverted in latest.

File: docs/config.rst (hunk 4 above)
Comment (tfurf) [reply docs/config.rst]: Fixed in latest.

**Pull request merged**